share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -204,6 +195,15 @@ default:
   extends: [ ".generate-base" ]
   tags: ["spack", "public", "medium", "neoverse_v2"]
 
+.generate-win64:
+  extends: [ ".base-job", ".generate-common" ]
+  before_script:
```

Comment:
I think this needs an empty after script?

Reply:
Added the equivalent for Windows